### DISCUSSION TITLE
AI發明的專利歸誰的?

### DISCUSSION THREAD

**Discussion by Raywei0214** [1101-GS3073-C/Python-Community-GS3073-C#589] [General]
這我覺得沒什麼好疑惑的，就是歸搞出這個AI的人，就像你用打蛋器把蛋給打散了，那但是誰打散的呢? 哈哈 是我啦
https://buzzorange.com/techorange/2022/09/14/ai-invention/

**Comment by Raywei0214** [comment]
論文的存在，令我無法停止對他的思考。論文因何而發生？我們一般認為，抓住了問題的關鍵，其他一切則會迎刃而解。

我以為我了解論文，但我真的了解論文嗎？仔細想想，我對論文的理解只是皮毛而已。

**Comment by lukiea** [comment]
專利這個問題就像是在打一場世界大戰，大家都在想分一杯羹，而不是真的為了這個世界而努力，希望世界能再出現根特斯拉一樣得人，不是為了個人利益，而是為了整個全體人類找想

**Comment by manipulateed** [comment]
我覺得專利應當歸發明人所有，其一他要為此負責任，而不是AI，其二，雖然AI創造出來的發明，是AI發明的，但終歸是人去訓練出來的人工智慧，自然需要提供訓練得過程與資料來源，履行人民之得權利。

**Comment by awkward-willy** [comment]
我曾在另一堂通識中聽老師探討相關議題，他就曾提到，當我們在探討權利時，與之相對的是義務與責任，而這也是現今各國之所以沒有辦法給予 AI 技術法人等資格的主要原因。畢竟當 AI 侵權等問題發生時，責任的歸屬會變得相當模糊，仍有待未來法規的規範。

**Comment by titan92601** [comment]
這是很有趣的問題。我認為是AI的，打蛋器打蛋是因為他本身的功能和模式就是打蛋。如今AI自己發明的我們人類都不想到的東西，它是有自主意識的。要比喻的話應該是我生了一個小孩他發明的東西專利應該是誰的。但現在人對於AI是否是人或是否擁有人權這都還沒有答案。所以先等他們把AI對於人的定義先發展出來再來討論這個問題吧！

**Comment by chia-kai-chang** [comment]
@Raywei0214 雖然看得出是你的原創文章，但是Discussion有3點，可以多寫一點墨水阿~ 多多告訴大家你的感想，心情，不用管政治正不正確阿，只希望大家多抒發，表達個人的觀點。

**Comment by limitx0** [comment]
完全不懂為什麼要給AI的創作申請專利
AI也沒有想法要去對他的創作做利用

**Comment by Yilin20021016** [comment]
這個問題是個好問題 如果AI有自我思想不要當人的專利時要如何?

**Comment by Aerofe** [comment]
我認為AI是不需要專利的，專利是維護發明者的權利，但AI目前還只是個輔助型的工具，並非為人
